### DISCUSSION TITLE
CT Attribute table field expansion

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5988] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

Stantec needs to store 60 fields worth of abiotic data. This data should be at the CT Attribute table level and apply to all CEs associated with the CT and COs thereafter. 

The [CT Attribute](https://files.specifysoftware.org/schema/version/2.10/#collectingtripattribute) table has 17 text fields, 10 number fields, 10 integer fields, and one remarks field available, leaving us 22 fields short. There aren't enough free fields on CT to hold the remaining 22 fields, but separating the data in this way is not ideal anyway. 

**Describe the solution you'd like**

We'd like at least a 22-field expansion to the CT Attribute table. The [CO Attribute](https://files.specifysoftware.org/schema/version/2.10/#collectionobjectattribute) table's schema is a sufficient template with more than enough fields available. 

**Describe alternatives you've considered**

- adding the data as an attachment is an option in theory, but they need to be able to export the data as a flat file, and prefer to have this queryable 
- splitting the data amongst other forms/tables is also not ideal for querying, nor is it sensical for their data structure (CT>CEs>COs)

**Reported By**
stantec

**Additional Context** 

<img width="319" alt="Screenshot 2024-10-17 at 1 21 44 PM" src="https://github.com/user-attachments/assets/675decec-753e-4db8-b39b-3eaeeae13cda">

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8207
